### PR TITLE
Ignore threaded replies

### PR DESCRIPTION
**Pull request by adamweeks**
This ignores any conversation activity that is a reply. It ignores the activities that come across via websocket as well as the activities that are loaded with the conversation.

### CODE REVIEW

File: packages/node_modules/@ciscospark/redux-module-conversation/src/helpers.js [hunk 1]
```diff
@@ -55,7 +57,22 @@ export function normalizeActivities(activities) {
...
-  return activities.filter((a) =>
-    !(a.verb === 'update' &&
-    a.object && a.object.objectType === 'content'));
+  return activities.filter((a) => {
+    const isUpdate = a.verb === API_ACTIVITY_VERB.UPDATE;
+    const isContent = a.object && a.object.objectType === 'content';
+    const isReply = a.type === API_ACTIVITY_TYPE.REPLY;
+    let shouldInclude = true;
+
+    // !(a.verb === 'update' &&
```
Comment (akoushke) [inline packages/node_modules/@ciscospark/redux-module-conversation/src/helpers.js]: ?

File: packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js [hunk 2]
```diff
@@ -69,6 +69,11 @@ function processActivity(activity, space, props) {
     spacesList
   } = props;
 
+  // Reply activities are not currently supported
+  if (activity.type === API_ACTIVITY_TYPE.REPLY) {
```
Comment (akoushke) [inline packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js]: is this quoting?

Comment (adamweeks) [reply packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js]: No, this is threaded replies. We probably need a feature epic for click quoting.

File: packages/node_modules/@ciscospark/redux-module-conversation/src/helpers.js (hunk 1 above)
Comment (adamweeks) [reply packages/node_modules/@ciscospark/redux-module-conversation/src/helpers.js]: oops, old code, will remove

**Comment by adamweeks** [comment]
@akoushke I've addressed your comments. Let me know if there are any others.

File: packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js (hunk 2 above)
Comment (akoushke) [reply packages/node_modules/@ciscospark/widget-recents/src/enhancers/listeners.js]: quoting kinda works actually. they are doing a mark ups with `>`.

**Comment by akoushke** [comment]
I don't know how to test this, since the feature is not released yet. can we wait on this and then test it after the feature is released?

**Comment by adamweeks** [comment]
> I don't know how to test this, since the feature is not released yet. can we wait on this and then test it after the feature is released?

No, we cannot wait because once they are enabled, it will break the widget. This is the same implementation the web client is doing.

**Comment by akoushke** [comment]
> > I don't know how to test this, since the feature is not released yet. can we wait on this and then test it after the feature is released?
> 
> No, we cannot wait because once they are enabled, it will break the widget. This is the same implementation the web client is doing.

where are they enabled?

**Pull request merged**